.travis.yml

```
@@ -84,7 +84,20 @@ jobs:
       virt: vm
       env:
         - CIBW_BUILD: cp310-manylinux_aarch64
-      install: python3 -m pip install cibuildwheel==2.4.0
+      install: python3 -m pip install cibuildwheel==2.9.0
+      script: |
+        cibuildwheel --output-dir wheelhouse
+        source ./tools/wheels/upload_wheels.sh
+        set_travis_vars
+        set_upload_vars
+        upload_wheels # Will be skipped if not a push/tag/scheduled build
+    - python: "3.11"
```

According to [this table](https://docs.travis-ci.com/user/languages/python/#python-versions) is the correct name is `3.11-dev`?
```suggestion
    - python: "3.11-dev"
```

---

> According to this table is the correct name is 3.11-dev

Didn't work on multibuild, but worth a try.